vtm/src/org/oscim/core/MapElement.java

```diff
@@ -31,6 +34,8 @@
  */
 public class MapElement extends GeometryBuffer {
 
+    public List<GeoPoint> geoPoints = new ArrayList<>();
```

Comment:
`MapElement` extends `GeometryBuffer`, so holds already and very efficiently the points.
Better use the existing infrastructure, like elsewhere in library, for optimal performance.